crate/operator/update_user_password.py

```diff
@@ -104,9 +110,12 @@ async def update_user_password(
             # representation of the exception.
             exception_logger("... failed. Status: %s Message: %s", e.status, e.message)
             raise TemporaryError(delay=config.BOOTSTRAP_RETRY_DELAY)
-        else:
-            if "ALTER OK" in result:
-                logger.info("... success")
-            else:
-                logger.info("... error. %s", result)
-                raise TemporaryError(delay=config.BOOTSTRAP_RETRY_DELAY)
+        except TemporaryError as e:
+            raise e
```

Comment:
```suggestion
        except TemporaryError:
            raise
```

The exception is implicit ;-)